webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const path = require("path");
+
+module.exports = {
+    entry: {
+        "copyCodeBlock": "./src/copyCodeBlock.js"
+    },
+    output: {
+        path: path.join(__dirname, 'dist'),
+        filename: "[name].js"
```

Comment:
I'd suggest reading [this](https://webpack.js.org/guides/author-libraries/).  We'll want to [externalize](https://webpack.js.org/guides/author-libraries/#externalize-lodash) `highlight.js`.

We didn't have to [expose the entry as a library](https://webpack.js.org/guides/author-libraries/#expose-the-library) also; we didn't have to do this with vapor because there's no exports, but since we're exporting something from the module, we have to.

```suggestion
        filename: "[name].js",
        library: 'copy-code-block',
        libraryTarget: 'commonjs'
    },
    externals: {
        'highlight.js': 'highlight.js'
```

Reply:
Yup, I spaced on externals. Thanks!